tests/test_config_command.rs

```diff
@@ -438,6 +438,36 @@ fn test_config_edit_repo_outside_repo() {
     "###);
 }
 
+#[test]
+fn test_config_get() {
+    let test_env = TestEnvironment::default();
+    test_env.add_config(
+        r###"
+    [table]
+    string = "some value 1"
+    int = 123
+    list = ["list", "value"]
+    "###,
+    );
+
+    let stdout = test_env.jj_cmd_success(test_env.env_root(), &["config", "get", "table.string"]);
+    assert_eq!(stdout, "some value 1"); // no trailing newline
+
+    let stdout = test_env.jj_cmd_success(test_env.env_root(), &["config", "get", "table.int"]);
+    assert_eq!(stdout, "123"); // no trailing newline
+
+    let stdout = test_env.jj_cmd_failure(test_env.env_root(), &["config", "get", "table.list"]);
+    insta::assert_snapshot!(stdout, @r###"
+    Config error: invalid type: sequence, expected a string
+    For help, see https://github.com/martinvonz/jj/blob/main/docs/config.md.
```

Comment:
nit: better to rephrase the error message. The type may be valid even if `config get` can't serialize it.